opensrp-pnc/src/main/java/org/smartregister/pnc/activity/BasePncFormActivity.java

```diff
@@ -77,7 +77,7 @@ public void initializeFormFragment() {
 
     protected void initializeFormFragmentCore() {
         BasePncFormFragment pncFormFragment = (BasePncFormFragment) BasePncFormFragment.getFormFragment(JsonFormConstants.FIRST_STEP_NAME);
-        getSupportFragmentManager().beginTransaction().add(com.vijay.jsonwizard.R.id.container, pncFormFragment).commit();
+        getSupportFragmentManager().beginTransaction().add(com.vijay.jsonwizard.R.id.container, pncFormFragment).addToBackStack(pncFormFragment.getArguments().getString(JsonFormConstants.STEPNAME)).commit();
```

Comment:
you can revert this change ... its no longer needed

Reply:
reverted.